packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerCtas.tsx

```diff
@@ -22,106 +18,62 @@ interface MomentTemplateBannerCtasProps {
 }
 
 export function MomentTemplateBannerCtas({
-    mainContent,
-    mobileContent,
+    mainOrMobileContent,
     onPrimaryCtaClick,
     onSecondaryCtaClick,
     onReminderCtaClick,
     primaryCtaSettings,
     secondaryCtaSettings,
 }: MomentTemplateBannerCtasProps): JSX.Element {
+    const { primaryCta, secondaryCta } = mainOrMobileContent;
```

Comment:
can see used to reduce usage of primaryCta/secondaryCta in code below (making code more readable), great!